src/System.CommandLine.Tests/ParsingValidationTests.cs

```diff
@@ -185,12 +192,17 @@ public void When_FromAmong_is_used_and_multiple_invalid_inputs_are_provided_the_
 
             var result = command.Parse("list --columns c1 c2");
 
-            // Currently there is no possibility for a single validator to produce multiple errors,
-            // so only the first one is checked.
+            result.Errors.Count.Should().Be(2);
```

Comment:
> Was a test added to pin the expected behavior when ReportError is called more than once?

@jonsequitur I've extended this test to cover this scenario. Is that enough?

Reply:
The test makes sense but the test name no longer matches the scenario.